src/parser/mod.rs

```diff
@@ -4453,6 +4487,9 @@ impl<'a> Parser<'a> {
                     break;
                 }
             }
+            if let Token::EOF = self.peek_nth_token_ref(0).token {
+                break;
+            }
```

Comment:
can we collapse this into above to use a match statement?
```rust
match &self.peek_nth_token_ref(0).token {
    Token::Word(n) if ...
    Token::Eof 
}
```

Reply:
Done 👍. I was probably trying to minimize the diff for review here